### DISCUSSION TITLE
Is primary selection supposed to work?

### DISCUSSION THREAD

**Discussion by Ryan7865** [labwc/labwc#1034] [Q&A] [answered]
Hi,
being new to Wayland and labwc, I am confused about the state of primary selection. I am using Wayland v1.22 and labwc v0.6.4 (Arch Linux). According to the labwc changelogs, support has been added already in v0.4.0, yet it does not work. Selecting text and then pressing the middle mouse button does nothing. Not inside a Wayland application such as foot, nor inside an X application such as urxvt, and (of course) also not between applications. I understand there are security concerns, so I suspected it needs to be activated, but there is no corresponding option in "rc.xml.all". Am I missing something? Should it just work?

**Comment by Ryan7865** [comment, marked as answer]
It now works. Sorry, but it was my bad:
I had the mouse buttons remapped in X with xinput, and completely forgot about it (my mouse has a separate middle and scroll wheel button).

**Reply by Consolatis** [reply]
Thanks for following up with the reason why it didn't work for you in the beginning.
